pkgs/build-support/rust/hooks/cargo-setup-hook.sh

```diff
@@ -36,6 +36,8 @@ cargoSetupPostUnpackHook() {
     @cargoConfig@
 EOF
 
+    substituteInPlace .cargo/config.toml --subst-var-by BUILD_TOP "${NIX_BUILD_TOP:-/build-top}"
```

Comment:
Should we default this to /build, for consistency?

Reply:
It would be a noop either way, right? Since if `$NIX_BUILD_TOP` isn't set, then our cwd likely won't be `/build` either.

I guess we can default it to `/build` just because of semantics -- I guess that's all you're suggesting?